src/config.h

```diff
@@ -32,9 +36,9 @@ class Config {
   bool cpp_demangle = true;
   bool lazy_symbolication = true;
   bool print_maps_on_exit = true;
-  bool unstable_macro = false;
-  bool unstable_map_decl = false;
-  bool unstable_import = false;
+  bool unstable_macro = true;
```

Comment:
See comment elsewhere. I'm into the change in spirit, I just think you can introduce a basic enum type and parser here and make this behavior explicit rather than relying on an implicit (it is actually set in the config) value.

e.g.
```
UnstableFeature unstable_macro = UnstableFeature::warn;
```

Reply:
Done.